modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBContainerV1Test.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers.containers;
+
+import org.influxdb.InfluxDB;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class InfluxDBContainerV1Test {
```

Comment:
instead of using `@ClassRule`. Let's use `try-with-resources` in each test method and create the container in there.